lib/app/routes/comments.rb

```diff
@@ -10,6 +10,9 @@ class Comments < Core
           redirect "/"
         end
         comment = CreatesComment.create(submission.id, current_user, params[:body])
+
+        comment.increment_five_a_day if comment.qualifying?
```

Comment:
Hm, interesting suggestion. That makes a lot more sense story-wise. Move the `increment` method to user and move testing for `increment` to user tests, but keep `qualifying` in comment?